pkg/daemon/update.go

```diff
@@ -406,12 +407,12 @@ func generateKargsCommand(oldConfig, newConfig *mcfgv1.MachineConfig) []string {
 	cmdArgs := []string{}
 	for _, arg := range oldConfig.Spec.KernelArguments {
 		if !newKargs[arg] {
-			cmdArgs = append(cmdArgs, "--delete=" + arg)
+			cmdArgs = append(cmdArgs, "--delete="+arg)
```

Comment:
May be worth splitting out formatting stuff in a prep patch.

Reply:
If it's not too much work, I agree :+1:

Reply:
My vscode is doing this automatically somehow, need to figure that out first lol